src/pathfinding_service/model/channel_view.py

```diff
@@ -2,8 +2,8 @@
 
 from eth_utils import is_checksum_address
 
-from pathfinding_service.config import DEFAULT_PERCENTAGE_FEE, DEFAULT_REVEAL_TIMEOUT
-from raiden.utils.typing import ChannelID
+from pathfinding_service.config import DEFAULT_REVEAL_TIMEOUT
+from raiden.utils.typing import ChannelID, FeeAmount
```

Comment:
I feel like using `FeeAmount` instead of using `TokenAmount` is going a bit too far regarding the detailed typing. We don't need a separate type for each variable. And since we can't use inheritance on `NewType`s, a function like `format_token_amount` will have an ugly signature or require type casts when called on `FeeAmount`. Not that much of a problem, but we should remember to not just think that more different types is always better.

Reply:
This is something coming from Raiden. I think there it makes a lot more sense to keep stuff correct in the core.

As for `format_token_amount`, a `Union` should be fine, right?

Reply:
Yes, but if we go too far it will be a union of
* TokenAmount
* MediationFeeAmount
* PFSFeeAmount
* MSRewardAmount
* ...

and steadily growing.

Not much of a problem for this PR, I just wanted to mention it before we've invented even more detailed types. Should we also switch to FeeAmount for PFS fees, or is it for mediation fees only?

Reply:
At least in the client the `FeeAmount` is special, and there are (currently) no types for `[MS|PFS]FeeAmount`.